.github/bump_chart_versions.sh

```diff
@@ -40,4 +40,5 @@ fi
 echo "update ${CHART} appVersion from ${APPVERSION} to ${LATEST}"
 bump_patch_version
 sed -i "s/^version: .*$/version: ${NEW_VERSION}/" "${CHARTFILE}"
-sed -i "s/^appVersion: .*$/appVersion: ${LATEST}/" "${CHARTFILE}"
\ No newline at end of file
+sed -i "s/^appVersion: .*$/appVersion: ${LATEST}/" "${CHARTFILE}"
+sed -i "s/${APPVERSION}/${LATEST}/" "${CHARTFILE}"
```

Comment:
I was concerned that there could be a possibility that `$APPVERSION` might match some other string, like if we start adding release notes to the artifacthub metadata. It's also not quoting the dots so they'll be wildcards in this match.

I was thinking about maybe using yq (or gojq) to do this, roughly like:
```
IMAGES=$(yq -r '.annotations["artifacthub.io/images"]' charts/console/Chart.yaml)
NEWIMAGES=$(yq -c --arg v $LATEST '(. | select(.name == "redpanda") | .image) |= "docker.redpanda.com/redpandadata/console:"+$v' <<< $IMAGES)
yq -Y --argjson v $NEWIMAGES '.annotations["artifacthub.io/images"] = $v' charts/console/Chart.yaml > temp
mv temp charts/console/Chart.yaml
```